From e29470d181f1cce983c07135dc332227cb00d475 Mon Sep 17 00:00:00 2001
From: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>
Date: Sat, 21 Sep 2024 13:45:49 -0400
Subject: [PATCH 5/5] Revert "firefox: only add Version = 2 on non-darwin"

launchd module now creates a service that will initialize gui app
environment with sessionVariables injected by modules. Firefox module
will inject MOZ_* variables needed to handle Version=2 in the
profiles.ini config now. This makes special handling for darwin here
unnecessary.

This reverts commit b5e09b85f22675923a61ef75e6e9188bd113a6e1.
---
 modules/programs/firefox/mkFirefoxModule.nix | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/modules/programs/firefox/mkFirefoxModule.nix b/modules/programs/firefox/mkFirefoxModule.nix
index a0681c40..51a4de25 100644
--- a/modules/programs/firefox/mkFirefoxModule.nix
+++ b/modules/programs/firefox/mkFirefoxModule.nix
@@ -61,8 +61,7 @@ let
     }) // {
       General = {
         StartWithLastProfile = 1;
-      } // lib.optionalAttrs (cfg.profileVersion == null) {
-        Version = cfg.profileVersion;
+        Version = 2;
       };
     };
 
@@ -343,13 +342,6 @@ in {
       };
     });
 
-    profileVersion = mkOption {
-      internal = true;
-      type = types.nullOr types.ints.unsigned;
-      default = if isDarwin then null else 2;
-      description = "profile version, set null for nix-darwin";
-    };
-
     profiles = mkOption {
       inherit visible;
       type = types.attrsOf (types.submodule ({ config, name, ... }: {
-- 
2.46.0

